app.js

```diff
@@ -1,14 +1,36 @@
 const express = require("express");
 
-const {getCategories}= require("./controllers/games");
+const {getCategories, getReviews, getReviewId}= require("./controllers/games");
 
 const app = express();
 
 app.get("/api/categories", getCategories);
+app.get("/api/reviews", getReviews)
+app.get("/api/reviews/:review_id", getReviewId)
 
 app.use((err, req, res, next) => {
-    console.log(err);
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  }else 
+  next(err);
+
+})
+
+
+app.use((err, req, res, next) => {
+  if (err.code === '22P02') {
+    res.status(400).send({ msg: 'Invalid input' });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
```

Comment:
Personally I would put you 500 error handler last in your app - it helps readability.